### DISCUSSION TITLE
24 hours work time and bot still hasn't found a match - any suggestions?

### DISCUSSION THREAD

**Discussion by Dimitah** [jef/streetmerchant#1062] [Help]
Hi there, 

It's obviously not a technical issue or anything like that. I was just wondering if it's normal for the bot not to find a single available listing for 24 or more hours of non-stop work. 

I am currently scraping for: 3070, 3080, rx6800, rx6800xt, rx6900xt, sf, sonyps5c, sonyps5de
In: amd, amazon-uk, corsair, currys, playstation
PAGE_SLEEP_MIN="2500"
PAGE_SLEEP_MAX="6000"

Any advises what I may need to add / change?

**Comment by Vearol** [comment]
@jef . @Dimitah , it just doesn't work, always prints "out of stock"
try adding some random valid item, which is actually in stock...
you can do it by modifying src/store/model/ ...

I added some random things with are in stock, but the tool keeps writing "Out of stock"

**Comment by Dimitah** [comment]
I was under the impression a lot of people managed to get what they were looking for using the bot. It surely works for some, this is why I was looking for a suggestion from someone. 

I've seen other people using whatever type of alert they chose to get cards - online stock checkers, Distill etc.

**Comment by Vearol** [comment]
indeed, I have seen on youtube stream, that it works...
but sadly it is not what I have found, after running with pre-configured items which are available and in stock

**Comment by Dimitah** [comment]
Well, this is why I posted here. I am hoping someone can come up with some suggestions, settings etc. I understand most people with a successful bot would prefer to keep quiet though, I probably would have :).

**Comment by Vearol** [comment]
what? ...no, why?)
if they are buying hardware with an intention to sell it for more money - then maybe
but otherwise - if a person just looking for 1 GPU for a personal computer like I do - then they have no reason not to share the story and help others to get a piece of this new hardware

**Comment by tavelm** [comment]
Check to make sure you're not IP banned from the seller.  Can you visit the URL you're scraping?

**Comment by Vearol** [comment]
@tavelm , yes just the URLs work fine

**Comment by jef** [comment]
Yes, this very possible. There are plenty of people that have gotten through the app. Even I have!

That being said, you can replace the links to see that the app is working correctly. All test series and models should work. Some of these items need updating as some of the 2060s are even sold out!

**Comment by Dimitah** [comment]
I have updated the bot yesterday for the last time, although I am not sure if links are being updated in the files or I need to manually update them.

**Comment by Vearol** [comment]
@jef , what about the test with items, that are in stock?
why does this app shows "out of stock" for them? and how to troubleshoot?

**Comment by thefl0yd** [comment]
I've had many near-misses so far. Last time because I was out walking the dog, but usually because I'm just not quick or lucky enough with many of these stores.

That said, I've had days go by with no matches. If you're looking for nvidia the releases are usually clustered. retailerA today, retailerB tomorrow or the day after, sometimes with retailerC and/or retailerD (...etc) on the same day, another day later, etc.

This is all based on my experience looking for an nvidia 3080 card in most of the US based stores. You may have a different experience. You may also be having another issue, but my point was it's not uncommon to go days without seeing anything other than the dreadful 'OUT OF STOCK'.

**Comment by DylanWritesCode** [comment]
Same issue here it appears the response const is always null within the lookupCard() function in lookup.ts

**Comment by DylanWritesCode** [comment]
Ok so this time I turned on headless and update the walmart model it works. However it only gets a response when the browser is not headless. I read some stuff it had to do with the module that is being used to communicate with the web
